addons/EditableWindow/src/presenter.js

```diff
@@ -1437,15 +1442,10 @@ function AddonEditableWindow_create() {
     //images are temporarily replaced with it's alt text wrapped in paragraph in purpose to getContent with text- this allows to avoid manual parsing HTML
     //after all, originalContent is being restored to editor
     presenter.getContentToRead = function EditableWindow_getContentToRead () {
-        const originalContent = presenter.configuration.editor.getContent();
-        const img = presenter.speechTexts.image;
-
-        let contentWithoutImages = presenter.configuration.editor.getContent().replace(/<img .*?alt="(.*?)".*?>/gm, `<p>${img} $1</p>`);
-        presenter.configuration.editor.setContent(contentWithoutImages);
-        let contentToRead = presenter.configuration.editor.getContent({format : 'text'});
-        presenter.configuration.editor.setContent(originalContent);
-
-        return contentToRead;
+        const rawHTMLContent = presenter.configuration.editor.getContent({format : 'raw'});
+        const regex = /<img .*?alt="(.*?)".*?>/gm;
+        const contentWithoutImages = rawHTMLContent.replace(regex, `<p>${presenter.speechTexts.image} $1</p>`);
```

Comment:
Jeżeli w `presenter.speechTexts.image` wrzucimy jakiś skrypt to wykona się on podczas czytania alt textów